spyro/solvers/time_integration_central_difference.py

```diff
@@ -277,9 +281,9 @@ def central_difference_MMS(Wave_object, source_id=0):
 
         B = fire.assemble(rhs, tensor=B)
 
-        Wave_object.solver.solve(X, B)
+        Wave_object.solver.solve(u_np1, B)
 
-        u_np1.assign(X)
+        # u_np1.assign(X)
```

Comment:
I would remove commented code. They make code harder to read and make it harder to search for word occurrences.

Reply:
Removed